src/frontend/screens/Onboarding/MapOnYourOwnIntro.tsx

```diff
@@ -55,13 +56,19 @@ export const MapOnYourOwnIntro = ({
   const {formatMessage: t} = useIntl();
   const {mutate: createProject, status} = useCreateProject();
   const {setActiveProjectId} = useActiveProjectIdActions();
+  const [allowNavigation, setAllowNavigation] = React.useState(false);
+
```

Comment:
You don't actually need this. Since there is conditional navigation logic happening you can just use `usePreventRemove(status === 'pending', () => {})` and it should work